scripts/status

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Get Umbrel Status
+
+if [[ $UID != 0 ]]; then
+    echo "Umbrel's status must be fetched as root"
+    echo "Please re-run this script as"
+    echo "  sudo ./scripts/status"
+    exit 1
```

Comment:
Is root required for this?

The current user should be added to the docker group so I think it should work without root.

And if they aren't in the docker group then docker should give them it's won warning.

Reply:
Ah okay, I just added it for consistency among all the three scripts.

Reply:
Will remove it, since it can be added later if root access is actually required.